docs/reference/gpu-arch/gpu-arch-spec-overview.rst

```diff
@@ -215,27 +647,54 @@ For a more detailed explanation refer to the :ref:`specific documents and guides
 LLVM target name
   Argument to pass to clang in `--offload-arch` to compile code for the given architecture.
 VRAM
-  Amount of memory available on the GPU.
+  Amount of memory available on the GPU. Some SKUs might have a different amount of VRAM.
 Compute Units
-  Number of compute units on the GPU.
-Wavefront Size
-  Amount of work-items that execute in parallel on a single compute unit. This is equivalent to the warp size in HIP.
+  Number of compute units (CUs) on the GPU.
+Warp Size
+  Amount of work-items that execute in a lock-step fashion.
+VGPR File
+  Size of the Vector General Purpose Register (VGPR) file. Holds data used in vector instructions.
+  GPUs with matrix cores also have AccVGPRs, which are Accumulation General Purpose Vector Registers, specifically used in matrix instructions.
+SGPR File
+  Size of the Scalar General Purpose Register (SGPR) file. Holds data used in scalar instructions.
 LDS
-  The Local Data Share (LDS) is a low-latency, high-bandwidth scratch pad memory. It is local to the compute units, shared by all work-items in a work group. In HIP this is the shared memory, which is shared by all threads in a block.
+  The Local Data Share (LDS) is a low-latency, high-bandwidth scratch pad memory. It is local to the WGP on RDNA GPUs and shared by the CUs, on CDNA and GCN GPUs it is local to the CUs. The LDS is shared by all work-items in a work group. In HIP this is the shared memory, which is shared by all threads in a block.
```

Comment:
```suggestion
  The Local Data Share (LDS) is a low-latency, high-bandwidth scratch pad memory. It is local to the WGP on RDNA GPUs and shared by the CUs. On CDNA and GCN GPUs it is local to each CU. The LDS is shared by all work-items in a work group. In HIP this is the shared memory, which is shared by all threads in a block.
```

> In HIP this is the shared memory, which is shared by all threads in a block.

I think its awkward to say it **is** the shared memory. It is _usually_ the physical memory storage backing for shared memory. Usually because the compiler is not *required* to use LDS for it, as it is usually not required to do anything in any particular way as long as it keeps to the language spec.

Reply:
Removed the glossary, as dicsussed, and added an additional (not yet working) reference to the HIP programming guide